server/sonar-web/src/main/js/components/issue/components/IssueCommentLine.js

```diff
@@ -46,6 +46,12 @@ export default class IssueCommentLine extends React.PureComponent {
     openPopup: ''
   };
 
+  handleCommentClick = (event /*: Event & {target: HTMLElement}*/) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)